### ISSUE TITLE
Trying to run locally, won't work 

### ISSUE THREAD

**Issue by nhershy** [PumpedSardines/Tajpi#5]
When removing the pollCheckForUpdate() function, I cannot get the app to work locally. I added program to accessibility. I see it running in the menu bar, but ctrl+key does not change to esperanto letter. Nor does the x or h method work. 

However, when I download your version and add it to accessibility , it works. 

However, I notice when I click the icon in the menu bar, the your version has different options than my local version. I am wondering if that is due to some background update yours gets. 

Can you help me figure out why running locally and removing the update code does not allow my version to work? 

Thanks :)
